packages/cli-repl/src/history.ts

```diff
@@ -0,0 +1,20 @@
+import redactInfo from 'mongodb-redact';
+
+/**
+ * Modifies command history array based on sensitive information.
+ * If redact option is passed, also redacts sensitive info.
+ *
+ * @param {array} History - Array of commands, where the first command is the
+ * most recent.
+ *
+ * @param {boolean} Redact - Option to redact sensitive info.
+ */
+function changeHistory(history: string[], redact: boolean = false) {
+  const hiddenCommands =
```

Comment:
Is case insensitive what we want? Also that could be a constant in the outer scope.

Reply:
hmmmm i am not sure. if the command is `Createuser` technically it's not a valid command and will error, but if someone goes in and types their password along with the wrong command I don't want it to be showing up in history either. so idk! what do you think?

Reply:
Don't know :). Not sure if something bad can happen if we get too liberal with that regex and we start to filter out safe input. For me is ok as is, but maybe max would have a better opinion.